server/core_friend.go

```diff
@@ -103,37 +111,47 @@ func AddFriends(logger *zap.Logger, db *sql.DB, currentUser uuid.UUID, ids []str
 		return err
 	}
 
-	// TODO(mo, zyro): Use notificationToSend to send notification here.
+	notifications := make(map[uuid.UUID][]*api.Notification)
+	for id, isFriendAccept := range notificationToSend {
+		uid := uuid.FromStringOrNil(id)
+		content, _ := json.Marshal(map[string]interface{}{"username": username})
```

Comment:
Let's define this once rather than in each loop iteration, since it doesn't change between notifications here.